lib/features/give/widgets/numeric_keyboard.dart

```diff
@@ -167,32 +170,29 @@ class _NumericKeyboardState extends State<NumericKeyboard> {
 
     var fontSize = 18.0;
     if (size.height < 600 && widget.presets.isNotEmpty) {
-      fontSize = 12;
+      fontSize = fontSize * 0.9;
     }
     final onButtonTap = onTap ?? () => widget.onKeyboardTap(value);
     final valueWidget = child ??
         Text(
           value,
           textAlign: TextAlign.center,
           style: Theme.of(context).textTheme.bodyMedium!.copyWith(
```

Comment:
The bodyMedium is pointless at this point as we use a fontSize, but it's still fine